src/components/Extensions/Details/DetailCard/Tabs/ParamList.tsx

```diff
@@ -21,36 +21,51 @@ import { Callout } from '../../../../common/Callout';
 import { DOCS_BASE } from '../../../../common/links/DocsLink';
 import { Markdown } from '../../../../common/Markdown';
 import { useExtension } from '../../../api/useExtension';
+import { isDynamicExtension } from '../../../api/useExtensions';
 import { EventsConfig } from './EventsConfig';
 import styles from './ParamList.module.scss';
 import { ParamValue } from './ParamValue';
 
 function ParamList() {
   const extension = useExtension()!;
+  const isDynamic = isDynamicExtension(extension);
+  const docSuffix = isDynamic
+    ? 'extensions/manage-installed-extensions?interface=sdk#reconfigure'
+    : 'extensions/manifest';
+
+  const dynamicFragment = (
+    <Typography use="body2">
+      Reconfigure is not available in the emulator. You can reconfigure
+      parameters by updating them in the code.
+    </Typography>
+  );
+
+  const staticFragment = (
+    <>
+      <Typography use="body2">
+        Reconfigure is not available in the emulator. You can reconfigure
+        parameters by updating your .env files with:
+      </Typography>
+      <br />
+      <code>firebase ext:configure {extension.id} --local</code>)
+    </>
+  );
+
   return (
     <div>
       <div className={styles.paramHeader}>
         <Callout aside type="note">
-          <Accordion
-            title={
-              <Typography use="body2">
-                Reconfigure is not available in the emulator. You can
-                reconfigure parameters by <b>updating your .env files</b> with:
-              </Typography>
-            }
-          >
-            <code>firebase ext:configure {extension.id} --local</code>
-            <div className={styles.learnButton}>
-              <a
-                href={DOCS_BASE + 'extensions/manifest'}
-                target="_blank"
-                rel="noopener noreferrer"
-                tabIndex={-1}
-              >
-                <Button>Learn more</Button>
-              </a>
-            </div>
-          </Accordion>
+          {isDynamic ? dynamicFragment : staticFragment}
+          <div className={styles.learnButton}>
+            <a
+              href={DOCS_BASE + docSuffix}
+              target="_blank"
+              rel="noopener noreferrer"
+              tabIndex={-1}
+            >
+              <Button>Learn more</Button>
```

Comment:
Hmm I don't think we should have buttons inside of anchors... that's not valid HTML